nobodywho/Cargo.toml

```diff
@@ -11,10 +11,10 @@ encoding_rs = "0.8.34"
 godot = { git = "https://github.com/godot-rust/gdext", branch = "master", features = [
     "experimental-threads",
 ] }
-llama-cpp-2 = { version = "0.1.83", features = ["sampler"]}
+llama-cpp-2 = { version = "0.1.84", features = ["sampler"]}
```

Comment:
I think we can remove the `sampler` feature now, right? As we removed that feature upstream.

Otherwise it looks good and its very nicely integrated with Godot!